pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -197,12 +197,13 @@ func (m *Multicluster) AddMemberCluster(clusterID cluster.ID, rc *secretcontroll
 		} else if features.WorkloadEntryCrossCluster {
 			// TODO only do this for non-remotes, can't guarantee CRDs in remotes (depends on https://github.com/istio/istio/pull/29824)
 			if configStore, err := createConfigStore(client, m.revision, options); err == nil {
-				m.remoteKubeControllers[clusterID].workloadEntryStore = serviceentry.NewServiceDiscovery(
+				m.remoteKubeControllers[clusterID].serviceEntryStore = serviceentry.NewServiceDiscovery(
```

Comment:
nit: while the type is technically serviceentry store, the name helps make it clear that this one only processes WorkloadEntry.

Reply:
ok, will revert the name unless once we support other types